packages/transactions/src/callHelpersContextParametrized.ts

```diff
@@ -131,3 +131,28 @@ export function createSendWithGasConstraints<A extends TxMeta, CC extends Contex
     );
   };
 }
+export function createSendWithGasConstraints1559<A extends TxMeta, CC extends ContextConnected>(
+  send: SendFunction<A>,
+  context: CC,
+  gasPrice$: GasPrice$,
+) {
+  return <B extends A>(callData: TransactionDef<B, CC>, args: B): Observable<TxState<B>> => {
+    return combineLatest(estimateGas(context, callData, args), gasPrice$).pipe(
+      first(),
+      switchMap(([gas, gasPrice]) => {
+        return createSendTransaction(send, context)(
+          {
+            ...callData,
+            options: (args1: B) => ({
+              ...(callData.options ? callData.options(args1) : {}),
+              gas,
+              maxPriorityFeePerGas: gasPrice,
```

Comment:
Shouldn't those arguments be separated somehow? So it will be: 
```
maxPriorityFeePerGas: someMaxPriorityFeePerGas
maxFeePerGas: someMaxFeePerGas
```

Reply:
In future they will be, but that is not a scope of this story 
in future  gasPrice$ will return two values